hudi-spark-datasource/hudi-spark-common/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -17,4 +17,4 @@
 
 
 org.apache.hudi.DefaultSource
-org.apache.spark.sql.execution.datasources.parquet.HoodieParquetFileFormat
\ No newline at end of file
+org.apache.spark.sql.execution.datasources.parquet.LegacyHoodieParquetFileFormat
```

Comment:
When switching to the new file format with the config, should the `NewHoodieParquetFileFormat` be registered too?

Reply:
Maybe? I'm not sure. What benefit does it give us?

Reply:
Just curious, I don't have a clear answer.  Since `createRelation` is overridden so functionality-wise it's ok.